lib/media.js

```diff
@@ -14,24 +16,16 @@ function flatMap(list, fn) {
 }
 
 /**
- * @param {Array} list
- * @param {string} name
- */
-function wrapComments(list, name) {
-  list.unshift('\n<!-- ' + name + ' Begin -->');
-  list.push('<!-- ' + name + ' End -->\n');
-}
-
-/**
  * @param {string} str
  * @returns {number}
  */
 function findBottom(str) {
-  let index = str.lastIndexOf('</body>');
+  var index = str.lastIndexOf('</body>');
```

Comment:
I accidentally deleted this line in one commit and then re-added it. I forgot that we can render one component if we use it's unique URL. 

But, also, there's no real need for `let` since `index` is scoped globally to this function. Using `let` doesn't actually serve a purpose in this case.